content/instructor/style-guide/what-to-teach/index.mdx

```diff
@@ -15,28 +15,28 @@ As with writing or drawing, instructing about _anything_ can be daunting. _Anyth
 
 Tell someone with writer’s block, “Describe your favorite food.” An hour later they’ll have an essay on cheeseburgers. Tell an artist, “Draw a cheeseburger,” and they’ll draw you a cheeseburger good enough to eat (and write about).
 
-![cheeseburger](../../../images/illustrations/ch2_s2_cheeseburger.png)
+![cheeseburger](../../../../src/images/illustrations/ch2_s2_cheeseburger.png)
```

Comment:
not sure if there's a preferred way to link to images. Reaching 4 folders up seems like a lot